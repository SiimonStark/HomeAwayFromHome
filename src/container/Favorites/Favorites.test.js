import React from 'react';
import {Favorites} from './index';
import {shallow} from 'enzyme';

describe('Favorites Container', () => {
  describe('Component', () => {
    let wrapper;
    beforeEach(() => {
      wrapper = shallow(<Favorites />)
    })
    it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    })
  })
})