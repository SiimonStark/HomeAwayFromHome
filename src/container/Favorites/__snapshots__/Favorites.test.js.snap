// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites Container Component should match snapshot 1`] = `
<section
  className="Job__display"
>
  <div
    className="Fav__container"
  >
    Favorites container
  </div>
</section>
`;
